### ISSUE TITLE
Accessibility labels in Proof request list are all "Proof Request Details"

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#2009]
### Description of problem
When voice control is enabled, when you enter your Proof request list, all proof request are labelled as "Proof request Details" which doesn't allow a person to tap on a particular proof request with their voice.

### Expected behavior
The accessibility label of each Proof Request should be the proof request name.

### Steps to reproduce
- enable voice control on iOS
- choose items names as voice control overlay
- go into BC Wallet proof request List
- try to tap on a particular proof request using only voice

### Environment
- [ ] Occurs on Android
- [x] Occurs on iOS

Build #: 1740

Android Device Model: 

iOS Device Model: iPhone SE

### Workaround
Enable item numbers overlay when configuring voice control

### Severity
- [ ] High
- [x] Medium
- [ ] Low

**Comment by jleach** [comment]
Should have been closed by stale bot but adding the label "needs/reproduction" reset. Closing.
